src/libScilla/ScillaIPCServer.cpp

```diff
@@ -22,9 +22,18 @@
 #include "libUtils/GasConv.h"
 #include "websocketpp/base64/base64.hpp"
 
+#include "libData/AccountData/Account.h"
 #include "libPersistence/BlockStorage.h"
 #include "libPersistence/ContractStorage.h"
 #include "libUtils/DataConversion.h"
+#include <algorithm>
+#include <cctype>
+#include <string>
+
+#ifndef __APPLE__
```

Comment:
Why do we need bits/stdc++?

Could you pls also move standard headers to the top of the file, and make local includes follow them?

Reply:
ah, accidentally included these